v2/pkg/runtime/window.go

```diff
@@ -163,6 +163,12 @@ func WindowIsNormal(ctx context.Context) bool {
 	return appFrontend.WindowIsNormal()
 }
 
+// WindowExecJS executes the given Js in the window
+func WindowExecJS(ctx context.Context, js string) {
```

Comment:
We are missing the implementation in the DevServer frontend. We could maybe embed the base Frontend into the DevServer so we don't need to always add a wrapper method there.

I have commit for that ready, if you want I can add it to the PR?

Reply:
That would be great 👍 Thank you 👍

Reply:
Just pushed it.